components/progressbar/stories/template.js

```diff
@@ -156,5 +169,30 @@ export const ProgressBarGroup = (args, context) => html`
 				${Sizes(args, context)}
 			</div>
 		</div>
+		<!-- The gradient story below supports linear-gradients used by Express. Adding trackFill or progressBarFill args
+		 to any template will set the --mod-progressbar-track-color or --mod-progressbar-fill-color custom properties. -->
+		 <div>
+			${Typography({
+				semantics: "heading",
+				size: "s",
+				content: ["Gradient support"],
+				customClasses: ["chromatic-ignore"],
+			}, context)}
+			<div style=${styleMap({
+				"display": window.isChromatic() ? "flex" : "none",
+				"flex-direction": "column",
+				"align-items": "flex-start",
+				"gap": "32px",
+				"border": "1px solid var(--spectrum-gray-200)",
+				"border-radius": "4px",
+				"padding": "12px",
+			})}>
+				${Template({
+					...args,
+					trackFill: "linear-gradient(to right, hotpink, orange)",
+					progressBarFill: "linear-gradient(to left, teal, purple)",
```

Comment:
Do we have "standardized" gradients or anything that I could use here instead? I saw on the `spectrum-two` branch that there's a specified gradient for `staticWhite`/`staticBlack` stories (https://github.com/adobe/spectrum-css/pull/2637/files), but I couldn't find anything like that in `main`.